### PR TITLE
Typo fix: Avater is now avatar

### PR DESCRIPTION
**Pull request by AvdLee**
Simple typo fix.

### CODE REVIEW

File: MockerTests/MockedData.swift [hunk 1]
```diff
@@ -11,7 +11,7 @@ import UIKit
 
 /// Contains all available Mocked data.
 public final class MockedData {
-    public static let botAvatarImageFileUrl: URL = Bundle(for: MockedData.self).url(forResource: "wetransfer_bot_avater", withExtension: "png")!
+    public static let botAvatarImageFileUrl: URL = Bundle(for: MockedData.self).url(forResource: "wetransfer_bot_avatar", withExtension: "png")!
```
Comment (ghost) [inline MockerTests/MockedData.swift:14]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/WeTransfer/Mocker/blob/5ea40997ceb09ebe9603bdc5098cfbc53ae51220/MockerTests/MockedData.swift#L14"></span>Force unwrapping should be avoided.</td>
    </tr>
  </tbody>
</table>

File: MockerTests/MockedData.swift [hunk 2]
```diff
@@ -11,7 +11,7 @@ import UIKit
...
     public static let exampleJSON: URL = Bundle(for: MockedData.self).url(forResource: "Resources/JSON Files/example", withExtension: "json")!
```
Comment (ghost) [inline MockerTests/MockedData.swift:15]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/WeTransfer/Mocker/blob/5ea40997ceb09ebe9603bdc5098cfbc53ae51220/MockerTests/MockedData.swift#L15"></span>Force unwrapping should be avoided.</td>
    </tr>
  </tbody>
</table>

File: MockerTests/MockedData.swift [hunk 3]
```diff
@@ -11,7 +11,7 @@ import UIKit
...
     public static let redirectGET: URL = Bundle(for: MockedData.self).url(forResource: "Resources/sample-redirect-get", withExtension: "data")!
```
Comment (ghost) [inline MockerTests/MockedData.swift:16]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/WeTransfer/Mocker/blob/5ea40997ceb09ebe9603bdc5098cfbc53ae51220/MockerTests/MockedData.swift#L16"></span>Force unwrapping should be avoided.</td>
    </tr>
  </tbody>
</table>

**Comment by ghost** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false"><a href="https://github.com/WeTransfer/Mocker/blob/5ea40997ceb09ebe9603bdc5098cfbc53ae51220/MockerTests/MockedData.swift#L22">MockerTests/MockedData.swift#L22</a> - Force tries should be avoided.</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">Executed 10 tests, with 0 failures (0 unexpected) in 4.889 (4.897) seconds</td>
    </tr>
  </tbody>
</table>

## Current coverage for Mocker is `94.62%`
✅ *No files affecting coverage found*

---

> Powered by [xcov](https://github.com/nakiostudio/xcov)

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Review by Boris-Em — APPROVED** [review]

**Pull request merged**